frontend/src/metabase/dashboard/components/DashboardInfoSidebar/DashboardInfoSidebar.unit.spec.tsx

```diff
@@ -0,0 +1,67 @@
+import userEvent from "@testing-library/user-event";
+import { renderWithProviders, screen } from "__support__/ui";
+import { createMockDashboard } from "metabase-types/api/mocks";
+import { setupRevisionsEndpoints } from "__support__/server-mocks/revision";
+import type { Dashboard } from "metabase-types/api";
+
+import { DashboardInfoSidebar } from "./DashboardInfoSidebar";
+
+interface SetupOpts {
+  dashboard?: Dashboard;
+}
+
+function setup({ dashboard = createMockDashboard() }: SetupOpts = {}) {
+  const setDashboardAttribute = jest.fn();
+  // TODO: Needs to change this name after https://github.com/metabase/metabase/pull/32357 is merged
+  const saveDashboardAndCards = jest.fn();
```

Comment:
Depending on which PR between https://github.com/metabase/metabase/pull/32357 and this PR is merged first. We need to update the function name accordingly.